octoprint_filamentsensorsimplified/__init__.py

```diff
@@ -144,13 +148,15 @@ def sensor_callback(self, _):
         filamentPresentInt = self.is_filament_present(self.setting_pin, self.setting_power, self.setting_triggered)
         if filamentPresentInt is 1:
             self._logger.info("Sensor was triggered")
+            self._event_bus.fire(Events.PLUGIN_FILAMENTSENSORSIMPLIFIED_CLOSED)
             if not self.changing_filament_initiated and self.printing:
                 self.send_out_of_filament()
             # change navbar icon to filament runout
             self._plugin_manager.send_plugin_message(self._identifier, dict(type="filamentStatus", noFilament=True,
                                                                             msg="Printer ran out of filament!"))
         elif filamentPresentInt is 0:
             self._logger.info("Sensor was not triggered")
```

Comment:
@valemaio2 I see you commented this out in master due to the log spamming issue. Did we figure out why that was being caused? I see this method is called `sensor_callback` - is it actually from the sensor calling back to the plugin or is it due to the plugin checking the GPIO too often?

My change here (line 151) is in the same place so if this PR is merged that might also cause spamming (I don't think I've seen the spamming locally, but I might have to do more testing)

Reply:
It could be because the plugin is checking the GPIO too often, in fact when it was spamming the log it was reporting the same state over and over again. I'm going to accept the PR, as I don't think it will cause spamming, but I'll keep an eye on it.